pager/src/main/java/com/google/accompanist/pager/Pager.kt

```diff
@@ -153,50 +100,35 @@ object PagerDefaults {
         state: PagerState,
         decayAnimationSpec: DecayAnimationSpec<Float> = rememberSplineBasedDecay(),
         snapAnimationSpec: AnimationSpec<Float> = SnapperFlingBehaviorDefaults.SpringAnimationSpec,
-        endContentPadding: Dp = 0.dp,
-        snapIndex: (SnapperLayoutInfo, startIndex: Int, targetIndex: Int) -> Int,
```

Comment:
in the previous api we had two overloads, the one without snapIndex and the one with snapIndex with no default value. in the new impl you left only one overload where snapIndex has a default value. it is not binary compatible change unfortunately because of how default params are generated in jvm. could you please keep both of the previous variants as deprecated, and then add two similar overloads(with and without snapIndex) in the new api shape without endContentPadding. this should help us in future to nicely migrate to the new snapping implementation we are currently working on in compose foundation. thanks!